docs/_config.yml

```diff
@@ -0,0 +1,39 @@
+title: coreos/fcct
+description: Fedora CoreOS Config Transpiler documentation
+baseurl: "/fcct"
+url: "https://coreos.github.io"
+# Comment above and use below for local development
+# url: "http://localhost:4000"
+permalink: /:title/
+
+remote_theme: pmarsceill/just-the-docs
```

Comment:
Should we be using a local fork?

Reply:
Luca mentioned that too. I will set this up as we will have more and more depending on that.